.buildkite/pipeline-utils/ci-stats/pick_test_group_run_order.ts

```diff
@@ -561,3 +561,50 @@ export async function pickTestGroupRunOrder() {
     ].flat()
   );
 }
+
+export async function pickScoutTestGroupRunOrder(scoutConfigsPath: string) {
+  const bk = new BuildkiteClient();
+  const envFromlabels: Record<string, string> = collectEnvFromLabels();
+
+  const rawScoutConfigs = JSON.parse(Fs.readFileSync(scoutConfigsPath, 'utf-8'));
+  const pluginsWithScoutConfigs: string[] = Object.keys(rawScoutConfigs);
+
+  const scoutGroups = pluginsWithScoutConfigs.map((plugin) => ({
+    title: plugin,
+    key: plugin,
+    usesParallelWorkers: rawScoutConfigs[plugin].usesParallelWorkers,
+    group: rawScoutConfigs[plugin].group,
+  }));
+
+  // upload the step definitions to Buildkite
+  bk.uploadSteps(
+    [
+      scoutGroups.length
+        ? {
+            group: 'Scout Configs',
+            key: 'scout-configs',
+            depends_on: ['build'],
+            steps: scoutGroups.map(
+              ({ title, key, group, usesParallelWorkers }): BuildkiteStep => ({
+                label: `Scout: [ ${group} / ${title} ] plugin`,
```

Comment:
This is a descriptive label for the Buildkite step, and I see it's working well.

<img width="531" alt="screenshot-bk-step-name" src="https://github.com/user-attachments/assets/6505a680-50e5-42dc-aea7-fa2dce87a6ea" />

I'm a fan!